Makefile

```diff
@@ -55,7 +44,3 @@ compiletime.h:
 clean:
 	$(MAKE) ARCH=$(ARCH) -C $(KDIR) M=$(PWD) clean
```

Comment:
```suggestion
	$(MAKE) -C $(KDIR) M=$(PWD) clean
```

Reply:
And this needs to be done above as well. If `CROSS_COMPILE` or `ARCH` are not set, e.g. on a native build, this might give us trouble.
Form first glance this should only set `CROSS_COMPILE` as empty which might work. But it is not correct.